src/file-utils.stanza

```diff
@@ -12,16 +12,35 @@ public defn open (f: (FileOutputStream) -> False, path: String, append?: True|Fa
   try: f(file)
   finally: close(file)
 
+
+doc: \<DOC>
+Convert Windows Backlash Path to Forward Slash Variant
+
+This is the reverse of the `norm-path` function in lbstanza core.
+This function is intended to help convert windows backslash paths
+into a consistent format (ie, linux/mac OS-X path format).
+<DOC>
+public defn un-norm-path (path:String) -> String :
```

Comment:
One possible improvement: some kind of wrapper type s.t. use of canonical paths is enforced by the typechecker

Reply:
Ya - Agreed. This almost feels like something we should be doing in `stanza` core. Handling paths like this in a OS independent way is such a common operation. 

I know there is a `ParsedPath` library in core. When I tried using this though - it didn't really solve the problem for me. There is no "Join" operation on `ParsedPath`. Seems like it needs more functionality a la [pathlib](https://docs.python.org/3.11/library/pathlib.html).